src/components/Home/Dashboard/Header/Menu/container.js

```diff
@@ -1,11 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 import { withRouter } from 'react-router-dom';
 import UserMenu from './component';
 import { authLogout } from '../../../../../store/authentifiction/actions';
 import { getUserLogin } from '../../../../../store/authentifiction/selectors';
 
-export const mstp = state => ({
+// eslint-disable-next-line no-shadow
+const MenuContainer = ({ authLogout, history, username }) => {
```

Comment:
Use class component and define `handleLogout` as method. Because this code will cause recreation of `handleLogout` function on each render. So we need to move it outside of render.